content/en/docs/setup/install/providers/kubernetes-v2/k8s-provider.md

```diff
@@ -207,7 +207,7 @@ hal config provider kubernetes account add my-k8s-account \
     --context $CONTEXT
 ```
 
-Finally, enable [artifact support](/docs/reference/artifacts/#enabling-artifact-support).
+Finally, enable [artifact support](/docs/reference/ref-artifacts/#enabling-artifact-support).
```

Comment:
Hmmm, this definitely fixes the link here, but I'm not sure this file is being used.  https://spinnaker.io/docs/setup/install/providers/kubernetes-v2/ already has the correct link, [here](https://github.com/spinnaker/spinnaker.io/blob/0ed61b4b520b72a545f91339c9d3af5359b149ad/content/en/docs/setup/install/providers/kubernetes-v2/_index.md?plain=1#L245).  How did you come across this?

Reply:
I came across it at https://spinnaker.io/docs/setup/install/providers/kubernetes-v2/k8s-provider/ which is a child page of https://spinnaker.io/docs/setup/install/providers/kubernetes-v2/ that you referenced.

Reply:
It looks like the current link point to the "Spinnaker Artifacts - Legacy UI" section instead of the "Spinnaker Artifacts" section. 
The only change I would suggest is changing the hard-coded url to use Hugo's `ref` shortcode so that Hugo will throw a compile error if the linked page moves in the future.

Something like: 
Finally, enable [artifact support]({{< ref "docs/reference/ref-artifacts#enabling-artifact-support" >}})

Reply:
Updated with suggested change.
Added co-authors.